radio/src/CMakeLists.txt

```diff
@@ -478,7 +478,7 @@ if(ARCH STREQUAL ARM)
     set(TARGET_SRC ${TARGET_SRC} haptic_driver.cpp)
   endif()
   add_definitions(-DCPUARM)
-  add_definitions(-DFRSKY -DFRSKY_SPORT -DFRSKY_HUB -DGPS -DPXX -DDSM2)
+  add_definitions(-DFRSKY -DFRSKY_SPORT -DFRSKY_HUB -DGPS -DPXX -DDSM2 -DMULTIMODULE)
```

Comment:
Please make MULTIMODE optional, see how it is done for other options, for example CLI.